site/gdocs/utils.ts

```diff
@@ -152,12 +153,20 @@ export const useLinkedChart = (
         }
     }
 
+    const parsedResolvedUrl = Url.fromURL(linkedChart.resolvedUrl)
+    const resolvedUrl = queryString
+        ? parsedResolvedUrl.setQueryParams({
+              ...parsedResolvedUrl.queryParams,
+              ...parsedOriginalUrl.queryParams,
+          }).fullUrl
+        : parsedResolvedUrl.fullUrl
+
```

Comment:
the decision to merge query params seems important enough to be documented, as it contrast with the old binary "use original params only if none present on the target" found in the WP code: https://github.com/owid/owid-grapher/blob/74009cda2266a7bde27c3531839de3f815c11dbe/baker/redirects.ts#L144-L149

I find the merge behaviour more difficult to reason with but ok if we have a good reason to introduce it.

Reply:
I'm on Martin's side here; merging query params is more reasonable to me than simply discarding them.

It at least gives the user a chance that, with an "incoming" query param of `?time=2010..2020`, they'll indeed end up seeing that time range in the target chart (it's not a guarantee as there might've been other changes when moving the chart to a mdim, like changing the default tab or sth like that).